### DISCUSSION TITLE
Component not being resolved during tests

### DISCUSSION THREAD

**Discussion by sf-cg** [nuxt/test-utils#733] [Q&A]
Hi,

I've been using the test-utils for a little bit now, and I'm adding it to a new nuxt project that uses a component library that gets installed as a plugin. However when I'm mounting the App in a test it's telling me it cannot resolve the component.

`[Vue warn]: Failed to resolve component: q-container`

These components work just fine when I actually run the app without any console log errors.

My plugin is installed via:

```js
export default defineNuxtPlugin((nuxtApp) => {
  nuxtApp.vueApp.use(q)
})
```

and the test:

```ts
import {describe, it, expect } from 'vitest'
import { mountSuspended } from '@nuxt/test-utils/runtime'
import App from '~/app.vue'

describe('Private pages', async (): Promise<void> => {

  it('protects private page', async (): Promise<void> => {
    const component = await mountSuspended(App, { route: '/private-page' })
    expect(component.html()).toContain('Must login')
  })

})
```

Has anyone else had this issue when using test-utils, or have any suggestions of what I can look into to resolve the issue?

**Comment by patrickyue** [comment]
add 
{ route: '/private-page' } =>
{ route: '/private-page',
  global: {
   q-container: 'your q-container here'
  }
 }
